go/keybase/main.go

```diff
@@ -93,6 +98,25 @@ func main() {
 	}
 }
 
+func tryToDisableProcessTracing(log logger.Logger, e *libkb.Env) {
+	if e.GetRunMode() != libkb.ProductionRunMode || e.AllowPTrace() {
+		return
+	}
+
+	// We do our best but if it's not possible on some systems or
+	// configurations, it's not a fatal error. Also see documentation
+	// in ptrace_*.go files.
+	if err := libkb.DisableProcessTracing(); err != nil {
+		log.Debug("Unable to disable process tracing: %v", err.Error())
+	}
+}
+
+func logStartupIssues(log logger.Logger) {
+	for _, err := range startupErrors {
+		log.Debug(err.Error())
```

Comment:
should this be `log.Error`?

Reply:
ah good call, it's only for that windows call right now and it was Errorf before (that didn't work because logging wasn't initialized at the time of the call).